### DISCUSSION TITLE
How many buses does the company currently own? 🚌🚍

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1265] [Galaxy Trivia] [answered]
The local bus company has recently expanded and no longer has enough space in its garage to accommodate all of its buses.

Twelve of their buses must be stored outside.

If they decide to increase their garage space by 40%, they will have enough room for all of their current buses, plus enough room to store twelve more in the future.

**Comment by tomerf** [comment, marked as answer]
The answer is 72.
The garage can store 60 buses, because a 40% increase will add room for 24 more buses, 12 that are currently outside and extra 12 in the future.

**Comment by peymansf2000** [comment]
The answer is 48.

**Reply by ParanoidUser** [reply]
Nice try!
